src/hooks/useAccommodationsAndOffers.tsx/index.ts

```diff
@@ -36,16 +58,48 @@ export const useAccommodationsAndOffers = (props: SearchTypeProps | void) => {
     }
   );
 
+  const latestQueryParams = data?.latestQueryParams;
+
   const allAccommodations = useMemo(
     () => normalizeAccommodations(data?.accommodations, data?.offers),
     [data]
   );
 
   // This includes accommodations with active offers.
-  const accommodations = useMemo(
-    () => allAccommodations.filter((a) => a.offers.length > 0),
-    [allAccommodations]
-  );
+  const accommodations = useMemo(() => {
```

Comment:
You could consider including an comment above `accommodation` stating what is happening here.

Also, I think we should eventually consider creating a helper function that those these transformations, so we keep the `index.ts` clean.  Something like

`accommodations = getNormalizedAccommodations(accommodations, latesQueryParams)`